core/assets/bundles/bundle_fr.properties

```
@@ -751,12 +751,12 @@ sector.curlost = Secteur perdu
 sector.missingresources = [scarlet]Ressources du Noyau insuffisantes !
 sector.attacked = Secteur [accent]{0}[white] attaqué !
 sector.lost = Secteur [accent]{0}[white] perdu !
-#note: the missing space in the line below is intentional
+#note: l'espace manquant dans la ligne ci-dessous est intentionnel
 sector.captured = Secteur [accent]{0}[white]capturé !
 sector.changeicon = Changer l'Icône
-sector.noswitch.title = Impossible de changer de Secteur
+sector.noswitch.title = Impossible de changer de secteur
```

```suggestion
sector.noswitch.title = Impossible de changer de Secteur
```
Le "S" est accentué dans la version anglaise donc il ne faut pas changer. https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L757